Bottom: a3c0d995a39723580641fe86f9081ebec7f615e5
Top:    79930dbaba2d52f920a3fc5c6eec33c7d5d4bf5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 11:14:12 +0100

Refresh of fix-grid-extents-in-3d-view

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 5b52b9a..0d1f7e0 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -328,7 +328,7 @@ ghid_make_gc (void)
   return rv;
 }
 
-void
+static void
 ghid_draw_grid (void)
 {
   static GdkPoint *points = 0;
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f40bce1..a5179ee 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -38,6 +38,8 @@ extern int ghid_gui_is_up;
 
 static hidGC current_gc = NULL;
 
+static void ghid_draw_grid (BoxTypePtr);
+
 /* Sets gport->u_gc to the "right" GC to use (wrt mask or window)
 */
 #define USE_GC(gc) if (!use_gc(gc)) return
@@ -609,7 +611,7 @@ ghid_make_gc (void)
   return rv;
 }
 
-void
+static void
 ghid_draw_grid (BoxTypePtr drawn_area)
 {
   static GLfloat *points = 0;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 600e109..0834ce6 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -491,8 +491,6 @@ gboolean ghid_pinout_preview_expose (GtkWidget * widget, GdkEventExpose * ev);
 GdkPixmap *ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int depth);
 hidGC ghid_make_gc (void);
 void ghid_destroy_gc (hidGC);
-//void ghid_draw_grid (void);                /* FIXME: GDK AND GL DIFFERENT */
-void ghid_draw_grid (BoxTypePtr drawn_area); /* FIXME: GDK AND GL DIFFERENT */
 void ghid_draw_bg_image (void);
 void ghid_use_mask (int use_it);
 void ghid_set_color (hidGC gc, const char *name);
